nym-vpn-core/nym-vpn-lib/src/lib.rs

```diff
@@ -47,3 +51,10 @@ pub use crate::{
         SpecificVpn,
     },
 };
+
+pub const DEFAULT_DNS_SERVERS: [IpAddr; 4] = [
```

Comment:
nitpick: separate config.rs file or similar?

Reply:
I'll do that in a separate PR to unblock people.